cli/Cargo.toml

```diff
@@ -72,6 +72,7 @@ log = { version = "0.4.14", features = ["serde"] }
 lspower = "1.1.0"
 notify = "5.0.0-pre.10"
 num_cpus = "1.13.0"
+once_cell = "1.8.0"
```

Comment:
This is already used in downstream dependencies and is being evaluated for the Rust standard library. It allows us to lazily create a value in a multi-threaded environment and then return a reference to that created value.